Gemfile

```diff
@@ -6,7 +6,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#   http://www.apache.org/licenses/LICENSE-2.0
+#   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please no: look at the content at the URL and see the end, the recommendation is to use the http URL. Changing this to https might break automatic license detection.